glusterd2/volume/struct.go

```diff
@@ -96,19 +96,20 @@ type Subvol struct {
 
 // Volinfo repesents a volume
 type Volinfo struct {
-	ID          uuid.UUID
-	Name        string
-	Type        VolType
-	Transport   string
-	DistCount   int
-	Options     map[string]string
-	State       VolState
-	Checksum    uint64
-	Version     uint64
-	Subvols     []Subvol
-	Auth        VolAuth // TODO: should not be returned to client
-	GraphMap    map[string]string
-	HealEnabled bool
+	ID                       uuid.UUID
+	Name                     string
+	Type                     VolType
+	Transport                string
+	DistCount                int
+	Options                  map[string]string
+	State                    VolState
+	Checksum                 uint64
+	Version                  uint64
+	Subvols                  []Subvol
+	Auth                     VolAuth // TODO: should not be returned to client
+	GraphMap                 map[string]string
+	HealEnabled              bool
+	GranularHealEntryEnabled bool
```

Comment:
Why can't this be a volume set ?

Reply:
Because before setting the option I need to run glfsheal binary with granular-entry-heal-op first and check if there are any heals pending.  I don't think volume set has the support for this.